### ISSUE TITLE
Polish Card component

### ISSUE THREAD

**Issue by trevordmiller** [eggheadio/egghead-ui#97]
Vojta has proposed some changes to the `Card` component in his latest styleguide mocks:
https://egghead.invisionapp.com/d/main#/console/10384631/225150686/comments/79600739

## Current

http://styleguide.egghead.io/?selectedKind=Card&selectedStory=Info&full=0&down=1&left=1&panelRight=0

Note that all this component currently does is render the `children` passed with a wrapper (background color white, a border radius depending on the `level`, and box shadow).

## Proposed changes

- Add an `overDark` option for a darker box shadow; default `overDark` to false

![image](https://cloud.githubusercontent.com/assets/5497885/24334108/a1574f3c-1221-11e7-9a20-ef1789402714.png)

**Comment by joelhooks** [comment]
@vojtaholik we want to lift **every** card with that strong of a shadow? So a stack of cards in a grid will have aggressive shadows? 

![image](https://cloud.githubusercontent.com/assets/86834/24334629/64497a74-1222-11e7-87f9-92329871a687.png)

I don't know that I'm digging that.

**Comment by vojtaholik** [comment]
@joelhooks no, I'll update this, It displays hover state atm.
@trevordmiller I'll update this issue and ping you once it's done.

**Comment by trevordmiller** [comment]
@vojtaholik Thanks 👍

**Comment by trevordmiller** [comment]
@vojtaholik Is this one ready to go?

**Comment by vojtaholik** [comment]
@trevordmiller yes, the behavior of cards in live version is fine (looking at https://egghead.io/browse/libraries/redux).

**Comment by trevordmiller** [comment]
@vojtaholik I guess I'm not sure what should be done for this story; what needs to be updated from the current Card component here: https://styleguide.egghead.io/components/Card (see the variations over light and dark between levels `1` and `2`)

**Comment by vojtaholik** [comment]
Sorry for confusing @trevordmiller 

I can't tell whether there's any shadow in styleguide view you've linked to. So just to be sure we're on a same track - I like how cards look in `egghead.io/browse` section. ;)

I've also updated invision styleguide with these as I promised (sorry it took so long): 
![image](https://cloud.githubusercontent.com/assets/25487857/25524048/2f93821a-2c09-11e7-9f7c-50fc449e2ac2.png)

But as I said, I don't insist on any changes, it's looking good. We have probably tackled some adjustments with Evgenyi some time ago, so that might be it. :)

Thanks!
